ci/rmt-run-feature-tests

```diff
@@ -11,6 +11,12 @@ group "synchronize SCC product/repositories"
   rmt-cli sync
 groupend
 
+group "install feature test dependencies"
+pushd "$SOURCE"
+  bundle install --with test
```

Comment:
Why did this work before when we did not bundle the test gems? Maybe this already worked because of `RAILS_ENV=test`?

Reply:
This is a good question where I have no answer for it, I was looking briefly into it why this is no necessary which only spawned the question, how was this even working before